cmd/vochaintest/vochaintest.go

```diff
@@ -105,6 +105,33 @@ func main() {
 		}
 	}
 
+	oracleKey := ethereum.NewSignKeys()
+	if len(*oraclePrivKey) > 0 {
+		if err := oracleKey.AddHexKey(*oraclePrivKey); err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		if err := oracleKey.Generate(); err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	treasurerKey := ethereum.NewSignKeys()
+	if len(*treasurerPrivKey) > 0 {
+		if err := treasurerKey.AddHexKey(*treasurerPrivKey); err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		if err := treasurerKey.Generate(); err != nil {
```

Comment:
I made this wrong, if the treasurer address is not provided in advance (genesis) the randomly generated key is not going to work as the genesis will take into account the hardcoded one

Reply:
fixed in https://github.com/vocdoni/vocdoni-node/pull/502/files#diff-6bff2e932680154a119e047cbfcaa19a9d11335ff65a7eccf163ac8a588ca1a4R173